### ISSUE TITLE
README preliminary evaluation

### ISSUE THREAD

**Issue by Rodolfo47** [cristoichkov/GBS_Bioinf_Process_Mamm#23]
Hi Cristian,
I think your general README.md and the structure of the repo are well
Your scripts a well commented and you have a good discussion .mb file

Your out/README.md is empty
Make sure you up date your class issues
I think that the out/R_plots/clust_tresh_multi.png can be great if yo use good font size and not to many 000s in the y values

well done!

**Comment by AliciaMstt** [comment]
Very nice and well documented README :)

Just few comments:

1. The first txt line of your readme says: "This repository contains a proposal of a workflow to process GBS data mainly with ipyrad (Fig. 1)" This doesn't seem to correspond to your repository title. You need to include somewhere as part of this parragraph that your data comes from Mammillaria haageana and what is your main objective. If in this repo you are only going to have the de novo assembly optimization, then also change your title to fit this objective. 

2. Your README doesn't explain in what *order* one has to run your scripts. Please add a section explaining the pipeline.

**Comment by cristoichkov** [comment]
I followed the recommendations for my repository.
1.- I deleted the out/README.md file
2.- I changed the font size of all my plots
3.- I modified the title, abstract and aims so that it corresponded with my repo. 
4.- I created a little workflow where I explain the pipeline.  

Thanks for their advice. 

I leave the link to my final project, I wrote it for you to review and give me your opinions and criticisms about it. 

[Final_project.md](https://github.com/cristoichkov/GBS_Bioinf_Process_Mamm/blob/master/Final_project.md)
